homeassistant/components/lastfm/sensor.py

```diff
@@ -58,7 +59,7 @@ def __init__(self, user, lastfm_api):
         self._unique_id = hashlib.sha256(user.encode("utf-8")).hexdigest()
         self._user = lastfm_api.get_user(user)
         self._name = user
-        self._entity_id = user
+        self._entity_id = slugify(user)
```

Comment:
Why don't we drop this? We have a unique ID, so Home Assistant will just register a new entity ID based on the name that is unique. Doing it like this gets an annoying entity ID

Reply:
Thanks for the feedback - won't using a "friendly" entity_id be more user friendly than a sha256 of the username though?

Reply:
Also would dropping this keep BC with previous people's setups with using the friendlier `my_username` kinda naming.

Reply:
If `entity_id` isn't set when adding the entity, the `name` property will be used as template for the entity_id.

Reply:
Interesting - so I could drop `entity_id` entirely?

Reply:
Yes, if we don't need to set `entity_id` to something else than `name`, which we rarely do, that's fine. :+1:

Reply:
Good to know - thanks for clarifying. Simplicity is better for sure!